src/test/java/ca/yorku/cse2311/tab2pdf/parser/DoubleBarParserTest.java

```diff
@@ -0,0 +1,110 @@
+package ca.yorku.cse2311.tab2pdf.parser;
+
+import ca.yorku.cse2311.tab2pdf.model.DoubleBar;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class DoubleBarParserTest {
+
+    /**
+     * None of these lines should parse into a Space
+     */
+    public static final String[] INVALID_LINES = {
+            "-----1-----1-----1-----1-|-----1-----1-----1-----1-|"
+            , "1-----1-----1-----1-|-----1-----1-----1-----1-|"
+            , "-|----1-----1-----1-|-----1-----1-----1-----1-|"
+            , "**||-----1-----1-----1-|-----1-----1-----1-----1-|"
+            , "|||3-----1-----1-----1-|-----1-----1-----1-----1-|"
+            , "|-----1-----1-----1-|-----1-----1-----1-----1-|"
+    };
+
+    /**
+     * All of these lines should parse into a space
+     */
+    public static final String[] VALID_LINES = {
+            "||*-----1-----1-----1-|-----1-----1-----1-----1-|"
+            , "*||----1-----1-----1-|-----1-----1-----1-----1-|"
+            , "*|5*---1-----1-----1-|-----1-----1-----1-----1-|"
+            , "*||*--1-----1-----1-|-----1-----1-----1-----1-|"
+            , "|2-1-----1-----1-|-----1-----1-----1-----1-|"
+            , "||-----------------------------"
+    };
+
+    public static final DoubleBar[] VALID_RESPONSES = {
+            new DoubleBar(1, true, false),
+            new DoubleBar(1, false, true),
+            new DoubleBar(5, true, true),
+            new DoubleBar(1, true, true),
+            new DoubleBar(2, false, false),
+            new DoubleBar()
+    };
+
+    private DoubleBarParser parser;
+
+    @Before
+    public void setUp() {
+        parser = new DoubleBarParser();
+    }
+
+    @Test
+    public void testParseInvalidLines() {
+
+        for (String line : INVALID_LINES) {
+            try { // Go through each invalid line, hoping to throw an Exception, since they cannot be parsed
+
+                try {
+                parser.parse(line);
+                } catch (NumberFormatException e) {
```

Comment:
This test should be split up into two. One test should expect a `ParseException`, and one should expect a `NumberFormatException`. These tests should fail if the wrong exception is thrown